hack/make/cross

```diff
@@ -27,6 +27,13 @@ for platform in ${DOCKER_CROSSPLATFORMS}; do
 			export GOARM
 		fi
 
+		# FIXME: gcc-multilib uninstalls gcc-arm compilers
+		# worksaround to install during runtime and build linux/386 last
+		# https://bugs.launchpad.net/ubuntu/+source/gcc-defaults/+bug/1300211
+		if [[ "${GOARCH}" = "386" ]]; then
+		    apt-get update && apt-get install -y --no-install-recommends gcc-multilib
```

Comment:
@lmbarros this package is required to build `linux/386` (not sure if we even need it tbh); it wrecks ARM compiler config, so I am installing at runtime and building this target last.